scripts/installer/README.txt

```diff
@@ -40,7 +40,7 @@ from conf/jhove:
 
 jhove.conf
 
-SOLR schema and config files, from conf/solr/7.3.0: 
+SOLR schema and config files, from conf/solr/7.2.1: 
```

Comment:
Ditto about being behind the "develop" branch.

Reply:
Merging the latest from "develop" should update this to Solr 7.3.0 so that the change doesn't show up in the diff.